packages/commonjs/src/index.js

```diff
@@ -75,6 +75,25 @@ export default function commonjs(options = {}) {
       ? (id) => options.ignore.includes(id)
       : () => false;
 
+  const getIgnoreTryCatchRequireStatementMode = (id) => {
+    let mode =
+      typeof options.ignoreTryCatch === 'function'
+        ? options.ignoreTryCatch(id)
+        : Array.isArray(options.ignoreTryCatch)
+        ? options.ignoreTryCatch.includes(id)
+        : options.ignoreTryCatch || false;
+
+    if (mode !== 'remove' && mode !== true) {
+      // default - do not ignore, convert as usual
+      mode = false;
+    }
+
+    return {
+      canConvertRequire: mode === false,
```

Comment:
For simplicity, this could be inlined: `canConvertRequire: mode !== 'remove' && mode !== true`

Reply:
Yeah, evolution :-)